pkg/validations/cluster.go

```diff
@@ -177,7 +177,7 @@ func clustersHaveEksaVersion(mgmtCluster, cluster *v1alpha1.Cluster) bool {
 		return false
 	}
 
-	if cluster.Spec.EksaVersion == nil && mgmtCluster.Spec.EksaVersion == nil {
+	if cluster.Spec.EksaVersion == nil || mgmtCluster.Spec.EksaVersion == nil {
```

Comment:
curious why this change? If it was a bug, could we add a test?

Reply:
This change isn't needed. I just made it when I was looking into having the controller set eksaVersion.